Grammatical adjustments of `writing templates` usage guide to resolve issue #9400 (#9401)
wagtail/wagtail: docs/topics/writing_templates.md

        models.py
```

> (The names "css", "js" etc are not important, only their position within the tree.)

Reviewer: ```suggestion
(The names "css", "js" etc aren't important, only their position within the tree.)
```

This seems arbitrary to me, "aren't" is perfectly understandable as-is.

> (The names "css", "js" etc are not important, only their position within the tree.)

Reviewer: ```suggestion
(The names "css", "js" etc aren't important, only their position within the tree.)
```

This seems arbitrary to me, "aren't" is perfectly understandable as-is.

This filter takes a chunk of HTML content and renders it as safe HTML in the page. Importantly, it also expands internal shorthand references to embedded images, and links made in the Wagtail editor, into fully-baked HTML ready for display.

> You should apply this only to fields using `RichTextField

Reviewer: ```suggestion
Only fields using `RichTextField` need this applied in the template.
```

Personally I find the original text more correct. I think it’s more useful to state the need so people can infer what to do, than tell them what to do.

### `pageurl`

> `pageurl` takes a Page object and returns a relative URL (`/foo/bar/`) if the Page object is within the same Site as the current page. If the Page object is not within the same Site as the current page, `pageurl` will return an absolute URL like: (`http://example.com/foo/bar/`).

Reviewer: The existing content seems fine to me. It’s not that useful to restate this is about `pageurl` since that’s the heading right before, and I don’t think making this content 2x the length improves clarity much.

```suggestion
Takes a Page object and returns a relative URL (`/foo/bar/`) if within the same Site as the current page, or absolute (`http://example.com/foo/bar/`) if not.
```
